src/de/grobox/liberario/TransportNetwork.java

```diff
@@ -57,6 +57,12 @@ public TransportNetwork setDescription(@Nullable String description) {
 		return this;
 	}
 
+	public TransportNetwork setDescription(@Nullable String description, @Nullable String networks) {
```

Comment:
Do we ever need to pass in `null`? If yes, we should keep the annotation. 

Is it ever a problem if we pass in `null`, but we never need to do it? If yes, it should have a `@NonNull` annotation.